spec/system/consumer/split_checkout_tax_incl_spec.rb

```diff
@@ -152,7 +152,9 @@
           login_as(user_outside_zone)
         end
 
-        it "will be charged tax on the order" do
+        it "will not be charged tax on the order" do
+          pending("WIP")
```

Comment:
@filipefurtad0 What work in progress are we waiting for here? Did you need to test more?

Reply:
This WIP is really about me understanding what does the app should do. We have several inconsistencies in the case of inclusive tax + user outside the taxable zone. In addition to the original issue, I've spotted
https://github.com/openfoodfoundation/openfoodnetwork/issues/9548 yesterday.